test/cpos/test_cpo_start.cpp

```diff
@@ -16,28 +16,28 @@
 
 #include <catch2/catch.hpp>
 #include <execution.hpp>
+#include <test_common/type_helpers.hpp>
 
 namespace ex = std::execution;
 
-struct my_oper {
+struct my_oper : non_movable {
   bool started_{false};
-
   friend void tag_invoke(ex::start_t, my_oper& self) { self.started_ = true; }
 };
 
-struct op_value {
+struct op_value /*: non_movable*/ {
```

Comment:
Are those comments intentional?

Reply:
Yeah, the test case fails if the types is immovable. I want to remember that this one is intentionally movable, so I left it commented.

Reply:
I would have preferred something along `/* intentionally movable */` but a comment is definitely better than none

Reply:
```suggestion
struct op_value /*: non_movable*/ { // Intentionally movable!
```